handler/src/main/java/com/networknt/handler/LightHttpHandler.java

```diff
@@ -76,6 +78,15 @@ default void setExchangeStatus(HttpServerExchange exchange, String code, final O
      * @param status error status
      */
     default void setExchangeStatus(HttpServerExchange exchange, Status status) {
+        // Wrap default status into custom status if the implementation of StatusWrapper was provided
```

Comment:
now everytime call this method will try to get this bean, maybe can move this wrapper to the class level